README.md

```diff
@@ -30,71 +30,88 @@ The Getting Started instructions can be found [here](GettingStarted.md).
+
+-----
+
+## Preliminary Discussion
+To appreciate the necessity of new DEX standards, it first important to understand the deficiencies of the current status-quo:
+
+### Current DEX Deficiencies  
+One consequence of centralized script addresses is the necessity for liquidity pools and LP providers as discrete entities. LPs are a common feature of many DEXes, coming with undesirable properties and corresponding sets of workaround "solutions". However, these workarounds have issues themselves, as explored here:
 
-### Liquidity Pools
-The most commonly used DEX design is liquidity pools which have the downsides of:
- 
-  1. Impermanent Loss
-  2. Loss of Delegation Control
-  3. Difficulty of Concurrency
+| Undesirable Property | Workaround "Solution" | Issues |
+| :--: | :--: | :--: |
+| Impermanent Loss | Yield Farming & Concentrated Liquidity | Medium - Long term unsustainability |
+| Incomplete or No Delegation Control | <ul><li>Asset pool fractionalization</li><li>Indirect Delegation via Governance Tokens</li></ul> | <ul><li>Unfair distribution of Governance tokens </li><li>Unavoidable centralization of stake (Major issue for Ouroboros)</li></ul>
+| Scaling Bottlenecks | Batchers, Execution/Routing Engines, and/or other middlemen | Middlemen can take advantage of their position between users and the protocol. Even if MEV is mitigated, more users --> more execution demand --> centralization of middlemen |
 
-Each of these downsides have their own corresponding "solution": yield farming, governance tokens, and batchers, respectively. However, these "solutions" have downsides of their own:
- 
-  1. The long term sustainability of yield farming is questionable.
-  2. Fairly distributing governance tokens is difficult.
-  3. Batchers are effectively middle-men that can take advantage of their position between users and the protocol.
+The key takeaway here (especially from the last point) is even 
 
-Even more concerning is that, even if governance tokens are fairly distributed, this doesn't change the fact that a liquidity pool can only be delegated to one stake pool. Fractionalizing liquidity pools helps but it still results in a more centralized PoS blockchain than if each user was able to delegate their assets independently.
+Of course, this is not an exhaustive list, but here is the key takeaway: **even if users control their own keys, for a DEX to achieve true scale the users must also control their own *addresses*.**
+
+
+if some workarounds *can* be effective, the underlying design *principles* are inadequate. Much like Bittorrent, and the CSL-CCL stack, **the best p2p protocols are ones that scale in proportion to the number of participants**. DEXes are no different. With this in mind, we can explore alternative design principles:
```

Comment:
I'm guessing this is the other half of the sentence before? I think you can just start here with "Much like Bittorrent..."

Reply:
Shouldn't the paragraph end with a period instead of a colon?